.github/DISSCUSSION_TEMPLATE/q-a.yml

```
@@ -0,0 +1,31 @@
+title: '🙏 Help: '
+labels: ['help wanted']
+body:
+  - type: textarea
+    attributes:
+      label: Summary
+      description: |
+        What do you need help with?
+
+        Womit brauchen Sie Hilfe?
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Additional information / Zusätzliche Informationen
+      description: |
+        Any code snippets, error messages, or dependency details that may be related? (`kolibri info`)
```

Das mit der CLI würde ich versuchen, einfacher und klarer zu machen:
```suggestion
        Any code snippets, error messages, or dependency details that may be related? (Please run `$ npx @public-ui/kolibri-cli info` and paste the result)
```